shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -191,19 +191,16 @@ const CodeEditor = createClass({
 		});
 
 		// Note: codeMirror passes a copy of itself in this callback. cm === this.codeMirror. Either one works.
-		this.codeMirror.on('change', (cm)=>{this.props.onChange(cm.getValue());});
-		this.updateSize();
-
 		this.codeMirror.on('change', (cm)=>{
 			this.props.onChange(cm.getValue());
-
 			this.state.widgetUtils.updateWidgetGutter();
 		});
 
-		this.codeMirror.on('gutterClick', (cm, n)=>{
-			const { gutterMarkers } = this.codeMirror.lineInfo(n);
+		this.updateSize();
 
-			if(!!gutterMarkers && !!gutterMarkers['widget-gutter']) {
+		this.codeMirror.on('gutterClick', (cm, n)=>{
+			// Open line widgets when 'widget-gutter' marker clicked
+			if(this.codeMirror.lineInfo(n).gutterMarkers?.['widget-gutter']) {
```

Comment:
Ah I've been trying to figure out how to do the elvis operator with bracket-notation for forever, sweet.